addPlant.php

```diff
@@ -0,0 +1,137 @@
+        <link rel="stylesheet" href="addPlant.css">
+
+        <title>Mr Fancy Plants</title>
+    </head>
+    <body>
+        <header>
+            <div class="logo">
+                <h1> Mr Fancy Plants </h1>
+                <p> Girl, pileas!</p>
+            </div>
+        </header>
+
+        <a class="PLANTBUTTON" href="index.php">Back Home</a>
+        
+        <form method="POST" class="add_plant">
+
+                <label for="newPlant">Plant Name</label>
+                <input type="text" name="newPlantName" />
+
+                <label for="newPlantFamily">Select the family</label>
+                <select name="newPlantFamily">
+                    <option value="0">--Plant Family--</option>
+                    <option value="4">Foliage Plants</option>
+                    <option value="1">Succlents and Cacti</option>
+                    <option value="3">Flowing Plants</option>
+                    <option value="2">Trailing or Hanging Plants</option>
+                </select>
+                
+                <label for="newPlantScientificName">Scientific Name</label>
+                <input type="string" name="newPlantScientificName" />
+                
+                <label for="newPlantImage">Image URL</label>
+                <input type="url" name="newPlantImage" />
+
+                <label for="newPlantImage">Description</label>
+                <input type="textarea" name="newPlantDescription" />
+                
+                <input type="submit" class="sumbitPlant"/>
+            </form>
+              
+                <?php
+                    if(isset($success)){
+                        if($success) {
+                            echo 
+                            "<div class='submit_success'>";
+                            "Your $inputtedPlantName was added to the plant collection";
+                            "</div>";
+                        } else { 
+                            echo "Sacrebleu - t did not work! Please ensure you completed all the fields";
+                        }
+                    }
+                ?>
+            </div>
+
+            <!-- <div class="slider_wrapper">
+                <div class="slider_inner">
+                    <div class="slider_left">
+                    </div>
+                    <div class="slider_container">
+                        <div class=" slider_images">
+                            <div>
+                                <?php
+                                    echo PlantViewHelper::displayAllPlants($plants);
+                                ?>
+                            </div>
+                            <img src="#" />
+                            <img src="#" />
+                            <img src="#" />
+                            <img src="#" />
+                            <img src="#" /> -->
```

Comment:
Please remove any commented out code, you can always get back to it with git if you find you need it later